src/Microsoft.TemplateEngine.Cli/LocalizableStrings.resx

```diff
@@ -619,4 +619,7 @@ Run 'dotnet {1} --show-aliases' with no args to show all aliases.</value>
   <data name="MissingTemplateContentDetected" xml:space="preserve">
     <value>Unable to locate the specified template content, the template cache may be corrupted. Try running 'dotnet {0} --debug:reinit' to fix the issue.</value>
   </data>
+  <data name="InvalidNameParameter" xml:space="preserve">
+    <value>Name cannot contain any of the following characters {0} or character codes {1}</value>
```

Comment:
This may look awkward when there are no printable characters (or no non-printable characters).

Reply:
These are computed from `Path.GetInvalidNameChars`, there's always at least one item in each bucket